LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-request-resources.https.html

```diff
@@ -90,7 +90,67 @@
       destination: 'script',
       message: `Script load (url:${actual_url})`
     };
-  frame.contentWindow.load_script_with_integrity(actual_url, integrity);
```

Comment:
This is a manual import, as importing the entire service-workers directory introduced too much unrelated noise